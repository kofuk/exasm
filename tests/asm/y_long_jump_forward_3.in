j @end
nop
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
@end lui r0, 5
