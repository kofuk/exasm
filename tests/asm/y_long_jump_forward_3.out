@00 11000000 01111100 // j 0x7C
@02 00000000 00000000 // nop
@04 00110000 00000001 // lui r0, 0x01
@06 11000000 00000010 // j 0x02
@08 00110000 00000001 // lui r0, 0x01
@0a 11000000 00000010 // j 0x02
@0c 00110000 00000001 // lui r0, 0x01
@0e 11000000 00000010 // j 0x02
@10 00110000 00000001 // lui r0, 0x01
@12 11000000 00000010 // j 0x02
@14 00110000 00000001 // lui r0, 0x01
@16 11000000 00000010 // j 0x02
@18 00110000 00000001 // lui r0, 0x01
@1a 11000000 00000010 // j 0x02
@1c 00110000 00000001 // lui r0, 0x01
@1e 11000000 00000010 // j 0x02
@20 00110000 00000001 // lui r0, 0x01
@22 11000000 00000010 // j 0x02
@24 00110000 00000001 // lui r0, 0x01
@26 11000000 00000010 // j 0x02
@28 00110000 00000001 // lui r0, 0x01
@2a 11000000 00000010 // j 0x02
@2c 00110000 00000001 // lui r0, 0x01
@2e 11000000 00000010 // j 0x02
@30 00110000 00000001 // lui r0, 0x01
@32 11000000 00000010 // j 0x02
@34 00110000 00000001 // lui r0, 0x01
@36 11000000 00000010 // j 0x02
@38 00110000 00000001 // lui r0, 0x01
@3a 11000000 00000010 // j 0x02
@3c 00110000 00000001 // lui r0, 0x01
@3e 11000000 00000010 // j 0x02
@40 00110000 00000001 // lui r0, 0x01
@42 11000000 00000010 // j 0x02
@44 00110000 00000001 // lui r0, 0x01
@46 11000000 00000010 // j 0x02
@48 00110000 00000001 // lui r0, 0x01
@4a 11000000 00000010 // j 0x02
@4c 00110000 00000001 // lui r0, 0x01
@4e 11000000 00000010 // j 0x02
@50 00110000 00000001 // lui r0, 0x01
@52 11000000 00000010 // j 0x02
@54 00110000 00000001 // lui r0, 0x01
@56 11000000 00000010 // j 0x02
@58 00110000 00000001 // lui r0, 0x01
@5a 11000000 00000010 // j 0x02
@5c 00110000 00000001 // lui r0, 0x01
@5e 11000000 00000010 // j 0x02
@60 00110000 00000001 // lui r0, 0x01
@62 11000000 00000010 // j 0x02
@64 00110000 00000001 // lui r0, 0x01
@66 11000000 00000010 // j 0x02
@68 00110000 00000001 // lui r0, 0x01
@6a 11000000 00000010 // j 0x02
@6c 00110000 00000001 // lui r0, 0x01
@6e 11000000 00000010 // j 0x02
@70 00110000 00000001 // lui r0, 0x01
@72 11000000 00000010 // j 0x02
@74 00110000 00000001 // lui r0, 0x01
@76 11000000 00001010 // j 0x0A
@78 00110000 00000001 // lui r0, 0x01
@7a 11000000 00000110 // j 0x06
@7c 00000000 00000000 // nop
@7e 11000000 00001010 // j 0x0A
@80 00000000 00000000 // nop
@82 11000000 00000010 // j 0x02
@84 00110000 00000001 // lui r0, 0x01
@86 11000000 00000010 // j 0x02
@88 00110000 00000001 // lui r0, 0x01
@8a 00110000 00000101 // lui r0, 0x05
